### ISSUE TITLE
[BUG] Loss of stiffness with high VBD cable damping

### ISSUE THREAD

**Issue by Johan-Friemann** [newton-physics/newton#2557]
### Bug Description

If a catenary with 0.0 `stretch_damping` and 1.0 `stretch_damping` is compared the shape of the second example displays a far larger arch. This should not be the case as the stiffness should remain unchanged.

Change `stretch_damping=0.0` to `stretch_damping=1.0` and compare the shapes.

### Reproduction Script

```python
import numpy as np
import warp as wp

import newton


class Simulation:
    def __init__(
        self,
        viewer,
        args=None,
    ):
        # Store viewer and arguments
        self.viewer = viewer
        self.args = args

        # Simulation cadence
        self.fps = 60
        self.frame_dt = 1.0 / self.fps
        self.sim_time = 0.0
        self.sim_substeps = 10
        self.sim_iterations = 5
        self.sim_dt = self.frame_dt / self.sim_substeps

        cable_radius = 0.012

        builder = newton.ModelBuilder()
        builder.rigid_gap = 0.05  # Default for all shapes

        # Set default material properties before adding any shapes
        # Default config will be used by plane and any shape without explicit cfg
        builder.default_shape_cfg.ke = (
            1.0e6  # Contact stiffness (used by plane)
        )
        builder.default_shape_cfg.kd = 5.0e-1  # Contact damping
        builder.default_shape_cfg.mu = 1.0  # Friction coefficient
        builder.default_shape_cfg.density = 0.1

        #builder.add_ground_plane()

        num_seg = 20
        length = 10.0
        seg_length = length/num_seg

        start_position = wp.vec3(0.0, 0.0, 5.0)
        start = builder.add_body(
            xform=wp.transform(p=start_position-wp.vec3(0.1,0.0,0.0), q=wp.quat_identity()),
            is_kinematic=True,
        )
        builder.add_shape_box(
            start,
            hx=0.1,
            hz=0.1,
            hy=0.1,
            cfg=newton.ModelBuilder.ShapeConfig(has_shape_collision=False),
        )

        end_position = wp.vec3(length+0.1, 0.0, 5.0)
        end = builder.add_body(
            xform=wp.transform(p=end_position, q=wp.quat_identity()),
            is_kinematic=True,
        )
        builder.add_shape_box(
            end,
            hx=0.1,
            hz=0.1,
            hy=0.1,
            cfg=newton.ModelBuilder.ShapeConfig(has_shape_collision=False),
        )

        points, quats = (
            newton.utils.create_straight_cable_points_and_quaternions(
                start_position,
                wp.vec3(1.0,0.0,0.0),
                length,
                num_seg,
            )
        )


        bodies, joints = builder.add_rod(
            positions=points,
            quaternions=quats,
            radius=0.1,
            bend_stiffness=10.0,
            bend_damping=0.0,
            stretch_stiffness=1.0e6,
            stretch_damping=0.0,
        )


        builder.add_joint_fixed(
            parent=start,
            child=bodies[0],
            parent_xform=wp.transform(wp.vec3(0.1, 0.0, 0.0), wp.quat_identity()),
            child_xform=wp.transform(
                wp.vec3(0.0, 0.0, 0.0), wp.quat_identity()
            ),
        )

        
        builder.add_joint_fixed(
            parent=end,
            child=bodies[-1],
            parent_xform=wp.transform(wp.vec3(-0.1, 0.0, 0.0), wp.quat_identity()),
            child_xform=wp.transform(
                wp.vec3(0.0, 0.0, seg_length), wp.quat_identity()
            ),
        )
        

        builder.color()
        self.model = builder.finalize()

        self.model.requires_grad = False

        # self.collision_pipeline = newton.CollisionPipeline(self.model)
        self.model.rigid_contact_max = 400_000

        self.sim_time_wp = wp.zeros(
            1, dtype=wp.float32, device=self.model.device
        )
        # Create VBD solver for rigid body simulation
        self.solver = newton.solvers.SolverVBD(self.model)

        # Initialize states and contacts
        self.state_0 = self.model.state()
        self.state_1 = self.model.state()
        self.control = self.model.control()
        self.contacts = self.model.contacts()
        self.viewer.set_model(self.model)

        # Optional capture for CUDA
        self.capture()

    def capture(self):
        """Capture simulation loop into a CUDA graph for optimal GPU performance."""
        if wp.get_device().is_cuda:
            with wp.ScopedCapture() as cap:
                self.simulate()
            self.graph = cap.graph
        else:
            self.graph = None

    def simulate(self):
        """Execute all simulation substeps for one frame."""
        for _substep in range(self.sim_substeps):
            self.state_0.clear_forces()

            # Apply forces to the model
            self.viewer.apply_forces(self.state_0)

            # Collide for contact detection
            self.model.collide(
                self.state_0,
                self.contacts,
                # collision_pipeline=self.collision_pipeline,
            )

            self.solver.step(
                self.state_0,
                self.state_1,
                self.control,
                self.contacts,
                self.sim_dt,
            )
            # Swap states
            self.state_0, self.state_1 = self.state_1, self.state_0

    def step(self):
        """Advance simulation by one frame (either via CUDA graph or direct execution)."""
        if self.graph:
            wp.capture_launch(self.graph)
        else:
            self.simulate()

        self.sim_time += self.frame_dt

    def render(self):
        """Render the current simulation state to the viewer."""
        self.viewer.begin_frame(self.sim_time)
        self.viewer.log_state(self.state_0)
        self.viewer.log_contacts(self.contacts, self.state_0)
        self.viewer.end_frame()


if __name__ == "__main__":
    # Parse arguments and initialize viewer
    viewer = newton.viewer.ViewerGL()

    # Create example and run
    simulation = Simulation(viewer)

    while viewer.is_running():
        simulation.step()
        simulation.render()
```

### System Information

Newton: 1.1.0
GPU: NVIDIA RTX 4080 Laptop
OS: nvidia/cuda:13.2.1-cudnn-devel-ubuntu24.04 Docker image
Python: 3.12.3
warp: 1.12.1
